Unable to generate Javadoc for Logger setter method

We are unable to generate javadoc for the following method / line in our code: https://github.com/AzureAD/microsoft-authentication-library-common-for-android/blob/dev/common4j/src/main/com/microsoft/identity/common/java/logging/Logger.java#L62

It seems we are using Lombok's experimental method annotations and they do not seem to play very well with the generating of the javadoc: https://projectlombok.org/features/GetterSetter

We get the following error:

`common\common4j\src\main\com\microsoft\identity\common\java\logging\Logger.java:62: error: cannot find symbol
    @Setter(onMethod_ = {@Synchronized})
            ^
  symbol:   method onMethod_()
  location: @interface Setter`

--- a/common4j/src/main/com/microsoft/identity/common/java/logging/Logger.java
+++ b/common4j/src/main/com/microsoft/identity/common/java/logging/Logger.java
@@ -59,13 +59,22 @@ public class Logger {
     @Accessors(prefix = "s")
     private static boolean sAllowPii = false;
 
-    @Setter(onMethod_ = {@Synchronized})
     @Accessors(prefix = "s")
     private static String sPlatformString = "";
 
     private static final ReentrantReadWriteLock sLoggersLock = new ReentrantReadWriteLock();
 
     private static final Map<String, ILoggerCallback> sLoggers = new HashMap<>();
+
+    /**
+     * Set the platform string to be used when generating logs.
+     *
+     * @param platformString the platform string to set
+     */
+    @Synchronized
+    public static void setPlatformString(String platformString) {
+        Logger.sPlatformString = platformString;
+    }
 
     /**
      * Enum class for LogLevel that the sdk recognizes.
@@ -410,7 +419,7 @@ public class Logger {
     /**
      * If applicable, log the discarded log.
      * This is applicable for testing only (IDetailedLoggerCallback is package-private).
-     * */
+     */
     private static void logDiscardedLogIfApplicable(String logMessage, ILoggerCallback callback, @NonNull String tag, @NonNull Logger.LogLevel logLevel, boolean containsPII) {
         if (callback instanceof IDetailedLoggerCallback) {
             ((IDetailedLoggerCallback) callback).discardedLog(tag, logLevel, logMessage, containsPII);

